django/contrib/admin/widgets.py

```diff
@@ -387,8 +387,15 @@ def __init__(self, rel, admin_site, attrs=None, choices=(), using=None):
         self.attrs = {} if attrs is None else attrs.copy()
 
     def get_url(self):
+        """
+        specify ForeignKey field in `AutocompleteJsonView`
+
+        `to_field` may not be a primary_key
+        """
+        to_field = getattr(self.rel, 'field_name', 'pk')
```

Comment:
`'pk'` as a string is too naive. The remote model could have a pk with a different name.
The correct way to get the pk name is: `model._meta.pk.name`